doc/architecture/decisions/0027-di-alignment.md

```diff
@@ -0,0 +1,51 @@
+# 27. DI Alignment
+
+[Next >>](9999-end.md)
+
+Date: 2023-006-16
+
+## Status
+
+Proposed
+
+## Context
+
+Due to our move into the DI/One Login teams, we want to align our whole approach as much as possible with their current
+patterns. This is because we want to make our system familiar and easy to access for support and other teams inside the
+DI space, and to make sure our security and resiliency standings align with theirs.
+
+This document is looking at approaches to realign our system and architecture, but not the deployment pipeline, with
```

Comment:
Do we need another thought about the deployment pipeline?

Reply:
I think the secure pipeline etc stuff should be a separate ADR once we know more about the actual process